### PR TITLE
Byte array to hex string conversion

### PR DESCRIPTION
**Pull request by lgrahl**
I even added tests! :)

### CODE REVIEW

**Comment by lgrahl** [comment]
Can you release this as `0.11.0` once merged?

**Comment by lgrahl** [comment]
@dbrgn Hasn't really worked. Other ideas how to fix Travis?

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +10,7 @@ matrix:
         EXECUTE_BUILD_DOCS=true
     - jdk: oraclejdk8
 before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -f  $HOME/.grade/caches/modules-2/modules-2.lock
```
Comment (dbrgn) [inline .travis.yml]: Why that change? Probably an accident, right?

Comment (lgrahl) [inline .travis.yml]: What the... I haven't done that. Will undo.

**Comment by lgrahl** [comment]
The CryptError? If it's JDK-dependent, it's likely due to a malformed message from the client, I guess.

**Comment by dbrgn** [comment]
Let's merge this for now.

**Pull request merged**